packages/cardano-services/src/Program/services/postgres.ts

```diff
@@ -51,6 +51,41 @@ export const getPoolWithServiceDiscovery = async (
   });
 };
 
+/**
+ * Creates a reconnectPoolClient :
+ * - use passed srv service name in order to resolve the port
+ * - make dealing with failover (re-resolving the port) opaque
+ * - use exponential backoff retry internally with default timeout and factor
+ * - intercept 'query' operation and handle connection errors runtime
+ * - all other operations are bind to pool object without modifications
+ *
+ * @returns pg.Pool instance
+ */
+export const getPoolClient = async (pgConnectionCreator: PgConnectionCreator, logger: Logger): Promise<PoolClient> => {
+  let poolClient = await pgConnectionCreator('fixme');
```

Comment:
`'fixme'`?

Reply:
Yes, pushing early to run in CI. I'm getting a error in the test global setup that I want to ensure is reproducible